mesa/space.py

```diff
@@ -537,8 +519,17 @@ def exists_empty_cells(self) -> bool:
         return len(self.empties) > 0
 
 
-class SingleGrid(Grid):
-    """Grid where each cell contains exactly at most one object."""
+class SingleGrid(_Grid):
+    """Rectangular grid where each cell contains exactly at most one agent.
+
+    Grid cells are indexed by [x][y], where [0][0] is assumed to be the
```

Comment:
While we are on it, we should recommend `[x, y]` instead.

Reply:
done!